monitor/dashboards.yaml

```diff
@@ -0,0 +1,11 @@
+# # config file version
+apiVersion: 1
+
+providers:
+ - name: 'Io teX'
```

Comment:
remove to space between Io and teX?

Reply:
Mainly, I think it’s better to look apart, but if it makes more sense, it’s still good.